audio/cries.asm

```diff
@@ -1,5 +1,46 @@
 INCLUDE "constants.asm"
 
-SECTION "audio/cries.asm", ROMX
+SECTION "audio/cries.asm@Cry Header Pointers", ROMX
+CryHeaderPointers::
+	dbw $3C, $57C5
```

Comment:
Can you add shims for each cry header so these can use `dba Cry_Nidoran_M`, etc?